images/dotnet-sdk/configs/latest.apko.yaml

```diff
@@ -10,6 +10,7 @@ contents:
     - aspnet-7-runtime
     - aspnet-7-targeting-pack
     - wolfi-baselayout
+    - ca-certificates-bundle
```

Comment:
maybe issue is with some other certificates in #540 - look this package already included above

Reply:
I think issue is with the `ca-certificates-bundle` but looks like this PR targeted `dotnet-sdk` instead of `dotnet-runtime` by mistake.

@tuananh